Firestore/core/src/firebase/firestore/remote/grpc_operation.h

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2018 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef FIRESTORE_CORE_SRC_FIREBASE_FIRESTORE_REMOTE_GRPC_OPERATION_H
+#define FIRESTORE_CORE_SRC_FIREBASE_FIRESTORE_REMOTE_GRPC_OPERATION_H
+
+#include <utility>
+
+#include <grpcpp/support/byte_buffer.h>
+
+namespace firebase {
+namespace firestore {
+namespace remote {
+
+/* Interface for an operation submitted to GRPC completion queue. */
+class GrpcOperation {
+ public:
+  virtual ~GrpcOperation() {
+  }
+
+  // The operation is expected to be put on the completion queue once this
```

Comment:
These are implementation comment markers. These should delimited with documentation comment markers (`/**` and `*/`).